packages/ui/src/components/va-select/VaSelect.demo.vue

```diff
@@ -11,6 +11,109 @@
       />
       <p>Value: {{ defaultSingleSelect.value }}</p>
     </VbCard>
+    <VbCard
+      title="Autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultSingleSelect.value"
+        class="mb-6"
+        :options="defaultSingleSelect.options"
+        autocomplete
+      />
+      <p>Value: {{ defaultSingleSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        :options="defaultMultiSelect.options"
+        multiple
+        autocomplete
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete with max visible options"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        :options="defaultMultiSelect.options"
+        :max-visible-options="2"
+        multiple
+        autocomplete
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Autocomplete with create new option"
+      style="width: 400px;"
```

Comment:
Can't we have ajax autocomplete example? (can be moved to another issue and out of this release if hard to do)

Reply:
Also moved to separate issue.